### PR TITLE
docs: added code block closing

### PR DESCRIPTION
**Pull request by Ghamza-Jd**
Closing the code block

Screen Shot 2022-09-29 at 11 31 18 PM

### CODE REVIEW

File: socketio/src/client/builder.rs [hunk 1]
```diff
@@ -234,6 +234,12 @@ impl ClientBuilder {
     ///     .expect("connection failed");
     ///
     /// // use the socket
+    /// let json_payload = json!({"token": 123});
```
Comment (1c3t3a) [inline socketio/src/client/builder.rs:237]: If you use the `json!` macro here, please also add the import `use serde_json::json` as well.

**Review by 1c3t3a — CHANGES_REQUESTED** [review]
Good catch! Thanks for the PR :) I just have one comment.

File: socketio/src/client/builder.rs (hunk 1 above)
Comment (Ghamza-Jd) [reply socketio/src/client/builder.rs:238]: Done, thanks!

**Review by 1c3t3a — APPROVED** [review]
LGTM!

**Pull request merged**